cms/src/test/java/uk/nhs/digital/apispecs/SwaggerCodeGenOpenApiSpecJsonToHtmlConverterTest.java

```diff
@@ -62,6 +62,25 @@ public void getHtmlForSpec_rendersCompleteOpenApiJsonAsHtml() {
         );
     }
 
+    @Test
+    public void getHtmlForSpec_rendersIncompleteOpenApiJsonAsHtml() {
```

Comment:
This test method targets a very specific aspect of the specification source so it would be good to express that in the methods' name. Simply saying 'incomplete' feels a bit too general and one has to look at the JSON file name to figure out that we specifically care about the `components` field missing.

How about naming it along the lines of `rendersSpecification_withoutFailure_whenComponentsPropertyIsAbsent`?

Reply:
Since we're changing the method name: nothing to do with the current story but I've just noticed that a number of method names in this class start with `getHtmlForSpec` - it appears that, at some point, we may have changed the name of the method these tests invoke but have forgotten to update names of methods in this class.

Would you mind updating the names of the remaining methods here? I think we can drop the `getHtmlForSpec` part altogether and stick to the pattern of `doesSomething_whenSomethingElse`.

Reply:
Thanks Rafal, you right.  The name of testing must be specific to the case that we are testing. I'll change it.